index.js

```diff
@@ -29,3 +39,15 @@
 
   module.exports = mOcKiNgCaSe;
 })();
+<<<<<<< HEAD
+
+// This function check if the input is an array of string
+
+function isAnArrayOfString(input) {
+  return input.every(function(currentValue) {
+    return typeof currentValue === 'string'
+  });
+
+}
+=======
```

Comment:
Also here, You can check about conflicts [here](https://githowto.com/resolving_conflicts)